lib/pacio_test_kit/interactions_test.rb

```diff
@@ -1,26 +1,119 @@
 module PacioTestKit
   module InteractionsTest
```

Comment:
Instead of having special conditions for different resource Type, we should try to make `update_and_validate_resource` method more general. Update it to take `resource_to_update`, `field`, and `field_values` as arguments. For each resource type, we should aim to update a non complex field. The only special case should be `Bundle`. For Bundle, since we are choosing to update the first resource in the bundle, the first resource is alway `Composition` according to the [`Bundle-composition`](https://build.fhir.org/ig/HL7/fhir-pacio-adi/SearchParameter-Bundle-composition.html) search parameter definition. So should skip if it's not. Validating the response metadata should not be different for `Bundle`.